easybuild/tools/include.py

```diff
@@ -170,12 +170,23 @@ def include_easyblocks(tmpdir, paths):
     _log.debug("Included generic easyblocks: %s", included_generic_ebs)
     _log.debug("Included software-specific easyblocks: %s", included_ebs)
 
-    # inject path into Python search path, and reload modules to get it 'registered' in sys.modules
+    # prepend new location to Python search path
     sys.path.insert(0, easyblocks_path)
-    reload(easybuild)
-    if 'easybuild.easyblocks' in sys.modules:
-        reload(easybuild.easyblocks)
-        reload(easybuild.easyblocks.generic)
+    fixup_namespace_packages(easyblocks_path)
```

Comment:
in the tests we're making sure `easybuild.easyblocks` and `easybuild.easyblocks.generic` are imported before fiddling with `sys.path` (see above); do we need to do that here too?